sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -521,6 +521,22 @@ def test_replace_map():
     N = M.replace(F, G, True)
     assert N == K
 
+def test_rot90():
+    """
+    Parameter k specifies the rotation
+    k is 0 when the Matrix is not rotated
+    k is +ve when the Matrix is rotated Clockwise
+    k is -ve when the Matrix is rotated Anti-Clockwise
+    |k| > 1 represent iterative rotations 
+    i.e. 
```

Comment:
You should add these docs in the function `rot90` in the main file under the Parameters section.

Reply:
Done mate ...